main.go

```diff
@@ -27,17 +27,21 @@ func main() {
 			Usage:   "Run a command",
 			Action: func(c *cli.Context) error {
 				var t Task
-				t.Run(c.Args())
+				if err := t.Run(c.Args()); err != nil {
+					return cli.NewExitError(err.Error(), 2)
+				}
 				return nil
 			},
 		},
 		{
 			Name:    "update",
 			Aliases: []string{"u"},
-			Usage:   fmt.Sprintf("update %v", Name),
+			Usage:   fmt.Sprintf("Update %v", Name),
```

Comment:
Command name is actually "update" all lower case.

Reply:
I wanted to make it similar to the other command's "usage", but you're right it can be confusing. Reverting.